src/host.cpp

```diff
@@ -244,7 +244,7 @@ bool Host::Loop()
 		if( console_ != nullptr )
 			console_->Draw();
 
-		if( settings_.GetOrSetBool( "cl_draw_fps", false ) )
+		if( settings_.GetOrSetBool( "cl_draw_fps", true ) )
```

Comment:
Let's avoid changing this. I think for most players it's not important to show FPS counter.